src/js/view/view.js

```diff
@@ -901,9 +901,7 @@ function View(_api, _model) {
         const width = _model.get('width');
         const height = _model.get('height');
         const styles = getPlayerSizeStyles(width);
-        if (isNumber(width)) {
```

Comment:
@DanFerrer does this revert some of your aspect ratio work from earlier? are there any considerations for the mobile case?

Reply:
@pajong it seems to me like the problem is a missing size constraint. If it appears too big on desktop, i would say we should enforce a specific width or a smaller max-width, instead of removing max-width

Reply:
@jnatalzia This shouldn't break anything I'm aware of. We already have size constraints for ```max-width``` in ```floatingplayer.less``` at ```400px```with the exception of mobile portrait where we set the ```max-width``` to ```none```